src/pages/settings/Payments/PaymentsPage.js

```diff
@@ -290,7 +290,7 @@ class PaymentsPage extends React.Component {
                                 !this.props.isSmallScreenWidth ? styles.sidebarPopover : '',
                             ]}
                         >
-                            {this.props.isSmallScreenWidth && (
+                            {(Platform.OS !== 'web' || this.props.isSmallScreenWidth) && (
```

Comment:
Please read the style guide. We don't do this. If you need to run something on native then created a index.native.js file.

Have you tested this on Ipad or Android Tab?

Reply:
Yes I test it . 

Do you think we need to  copy the file twice one for native and one for web for this small change

Reply:
@parasharrajat  yes test it and work fine on iPad . 
I think of doing this change. 
```suggestion
                            {(this.state.anchorPositionTop || this.props.isSmallScreenWidth) && (
```

anchorPositionTop is 0 sense model come from down in case this value with 0 or issmalScreen true .

Reply:
this is the new change I mention from iPad 

![Screen Shot 2022-02-05 at 12 46 50 AM](https://user-images.githubusercontent.com/21364901/152613351-fc5c0afa-3337-4bb3-b695-4d748a30ce50.png)

Reply:
Do you think we can use `props.isMediumScreenWidth` here?

Reply:
not work with landscape on the tablet. 

we need a Paypal account to appear when the Popover and in native it come from a button in all cases. 
on web when screen is small only.

so that is why I go with this change

Reply:
https://github.com/Expensify/App/blob/8e6e31a20c13d103c3bdfe1a21b00afc98345ab1/src/libs/getClickedElementLocation/index.native.js#L2

Reply:
Ok. But depending on this condition is a hack. We should not depend on unrelated props. it can break easily. 

If you want to show this on the native and Mobile web, then create a utility like `canFocusInputOnScreenFocus`.  

a Name could be `ShouldShowOnBottomDockedPopover`

Reply:
it's not unrelated props if you change the this.state.anchorPositionTop to not be 0  then 
the popover will not be from the bottom and then we will need to disable the Paypal account view. 
like on the web here. 

![Screen Shot 2022-02-04 at 10 00 59 PM](https://user-images.githubusercontent.com/21364901/152595435-75a90e3b-153f-408e-bb2f-841391ae87cc.png)

Reply:
Ok. I don't mind this as well. Leaving this to @thienlnam.

Reply:
I don't mind this solution, but can we get a better comment to clarify what this condition means? Variable name alone is not clear enough for me

Reply:
@thienlnam  yes , i add it now